apm-sniffer/bytebuddy-patch/src/main/java/net/bytebuddy/agent/builder/SWDescriptionStrategy.java

```diff
@@ -0,0 +1,335 @@
+        TYPE_CACHE.put(Boolean.class, new TypeDescription.ForLoadedType(Boolean.class));
+        TYPE_CACHE.put(Byte.class, new TypeDescription.ForLoadedType(Byte.class));
+        TYPE_CACHE.put(Short.class, new TypeDescription.ForLoadedType(Short.class));
+        TYPE_CACHE.put(Character.class, new TypeDescription.ForLoadedType(Character.class));
+        TYPE_CACHE.put(Integer.class, new TypeDescription.ForLoadedType(Integer.class));
+        TYPE_CACHE.put(Long.class, new TypeDescription.ForLoadedType(Long.class));
+        TYPE_CACHE.put(Float.class, new TypeDescription.ForLoadedType(Float.class));
+        TYPE_CACHE.put(Double.class, new TypeDescription.ForLoadedType(Double.class));
+        TYPE_CACHE.put(void.class, new TypeDescription.ForLoadedType(void.class));
+        TYPE_CACHE.put(boolean.class, new TypeDescription.ForLoadedType(boolean.class));
+        TYPE_CACHE.put(byte.class, new TypeDescription.ForLoadedType(byte.class));
+        TYPE_CACHE.put(short.class, new TypeDescription.ForLoadedType(short.class));
+        TYPE_CACHE.put(char.class, new TypeDescription.ForLoadedType(char.class));
+        TYPE_CACHE.put(int.class, new TypeDescription.ForLoadedType(int.class));
+        TYPE_CACHE.put(long.class, new TypeDescription.ForLoadedType(long.class));
+        TYPE_CACHE.put(float.class, new TypeDescription.ForLoadedType(float.class));
+        TYPE_CACHE.put(double.class, new TypeDescription.ForLoadedType(double.class));
+    }
+
+    private AgentBuilder.DescriptionStrategy delegate = Default.HYBRID;
+
+    private String nameTrait;
+
+    public SWDescriptionStrategy(String nameTrait) {
+        this.nameTrait = nameTrait;
+    }
+
+    @Override
+    public boolean isLoadedFirst() {
+        return true;
+    }
+
+    @Override
+    public TypeDescription apply(String name,
+                                 @MaybeNull Class<?> type,
+                                 TypePool typePool,
+                                 AgentBuilder.CircularityLock circularityLock,
+                                 @MaybeNull ClassLoader classLoader,
+                                 @MaybeNull JavaModule module) {
+        // find from type cache
+        if (type != null) {
+            TypeDescription typeDescription = TYPE_CACHE.get(type);
+            if (typeDescription != null) {
+                return typeDescription;
+            }
+        }
+        // wrap result
+        return new SWTypeDescriptionWrapper(delegate.apply(name, type, typePool, circularityLock, classLoader, module), nameTrait);
+    }
+
+    /**
+     * A TypeDescription wrapper to remove fields, methods, interface generated by SkyWalking.
+     */
+    static class SWTypeDescriptionWrapper extends TypeDescription.AbstractBase implements Serializable {
+
+        /**
+         * The class's serial version UID.
+         */
+        private static final long serialVersionUID = 1L;
```

Comment:
```suggestion
        /**
         * The class' serial version UID.
         */
        private static final long serialVersionUID = 1L;
```

Is the serialVersionUID required by some places?